app/controllers/api/journeys_controller.rb

```diff
@@ -145,6 +145,7 @@ def create_event
         notes: 'Automatically generated event',
         details: {},
         supplier_id: supplier.id,
+        created_by: user_for_paper_trail || doorkeeper_application_owner&.name,
```

Comment:
To avoid duplication of the created by logic, maybe we can move this into a helper method which consolidates this logic, maybe in the api controller or anywhere more relevant with something like this:
```
def created_by
  user_for_paper_trail || doorkeeper_application_owner&.name || 'unknown'
end
```
Then we can reuse this everywhere?

Reply:
Can you also add a test to check that the journey events set by the controller will have the created at?

Reply:
Have moved it to the api controller, couldn't think of a better place either, and definitely knew I should move it even as I was creating the PR :D

Reply:
Added tests for the journey events controllers as well :)